src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -156,14 +157,14 @@ const styles = StyleSheet.create({
   username: {
     fontFamily: fonts.inter[700],
     fontSize: 12,
-    color: '#828282',
+    color: COLORS.blackgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with dynamic colors from a theme object improves modularity and consistency. This is a good practice.
```diff
-    color: '#828282',
+    color: COLORS.blackgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->